docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/glossary/GlossaryApp.js

```diff
@@ -4,7 +4,7 @@
  */
 
 import initializeGlossaryApp from 'Libraries/glossary/glossary';
-import './GlossaryApp.scss';
+// import './GlossaryApp.scss';
```

Comment:
Is this meant to be removed?  Or is this a "Trying something and forgot to uncomment it?"

Reply:
yes, if left in sass compilation stops dead in its tracks and the build will fail.